src/App.js

```diff
@@ -29,8 +29,25 @@ class App extends Component {
 
   showBucketList = (event) => {
     event.preventDefault();
+    const cachedBucketListKeys = localStorage.getItem('bucketList')
```

Comment:
This might throw an error if there are no items in 'bucketList'. Might want to include a conditional statement here to check localStorage.hasOwnProperty('bucketList') before assigning variable.

Reply:
Good catch thank you! I overlooked that scenario, but makes sense that it would throw an error if there isn't anything in localStorage for that item.